synapse/lib/stormlib/stix.py

```diff
@@ -483,8 +510,38 @@ def _validateConfig(core, config):
                         mesg = f'STIX Bundle config has invalid rel entry {formname} {stixtype} {stixrel}.'
                         raise s_exc.BadConfValu(mesg=mesg)
 
+def _validateStixProc(bundle, loglevel):
+    '''
+    Multiprocessing target for stix validation
+    '''
+    # This logging call is okay to run since we're executing in
+    # our own process space and no logging has been configured.
+    s_common.setlogging(logger, loglevel)
```

Comment:
Still ok when executing from a pool?

Reply:
We could actually do this during process initialization.

Reply:
this could easily be moved to a init function